src/scales/interpolatedColorScale.ts

```diff
@@ -14,6 +14,7 @@ export module Scale {
    * By default it generates a linear scale internally.
    */
   export class InterpolatedColor extends AbstractScale<number, string> {
+    protected _d3Scale: D3.Scale.QuantitativeScale;
```

Comment:
I assume this is to shadow the super class's variable.  If so... I am not a fan of this convention, if the super class changes this variable name, this class will not be able to detect the change other than through obscure bugs etc.  Probably should cast where applicable.

Also, I don't see why you need to make this change to begin with?  Nothing so far seems to require the fact that this is a QuantitativeScale